From: John Zupin <johnz@lunarg.com>
Date: Thu, 4 Jun 2020 20:22:43 -0600
Subject: packaging: patch as jammy version

---
 project_version.h.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/project_version.h.in b/project_version.h.in
index 7c7b4a6..51e9780 100644
--- a/project_version.h.in
+++ b/project_version.h.in
@@ -27,9 +27,9 @@
 #define GFXRECON_PROJECT_VERSION_MINOR @GFXRECONSTRUCT_PROJECT_VERSION_MINOR@
 #define GFXRECON_PROJECT_VERSION_PATCH @GFXRECONSTRUCT_PROJECT_VERSION_PATCH@
 
-#define GFXRECON_PROJECT_VERSION_DESIGNATION  "@GFXRECON_PROJECT_VERSION_DESIGNATION@"
-#define GFXRECON_PROJECT_VERSION_SHA1         "@GFXRECON_PROJECT_VERSION_SHA1@"
-#define GFXRECON_PROJECT_BUILD_TYPE           "$<CONFIG>"
+#define GFXRECON_PROJECT_VERSION_DESIGNATION  ""
+#define GFXRECON_PROJECT_VERSION_SHA1         "jammy"
+#define GFXRECON_PROJECT_BUILD_TYPE           ""
 
 #define GFXRECON_VERSION_STR_EXPAND(x) #x
 #define GFXRECON_VERSION_STR(x) GFXRECON_VERSION_STR_EXPAND(x)
