From: John Zupin <johnz@lunarg.com>
Date: Thu, 25 Feb 2021 16:39:54 -0700
Subject: patch: build info summary for bionic

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e4bcc31..786918b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,7 +42,8 @@ set(GFXRECONSTRUCT_PROJECT_VERSION_MAJOR 0)
 set(GFXRECONSTRUCT_PROJECT_VERSION_MINOR 9)
 set(GFXRECONSTRUCT_PROJECT_VERSION_PATCH 10)
 
-set(GFXRECON_PROJECT_VERSION_SHA1 "unknown-build-source")
+set(GFXRECON_PROJECT_VERSION_DESIGNATION "")
+set(GFXRECON_PROJECT_VERSION_SHA1 "focal")
 
 option(BUILD_STATIC "Build static binaries for HPC clusters (will not build the tracer library)" OFF)
 
