From: John Zupin <johnz@lunarg.com>
Date: Thu, 25 Feb 2021 16:38:38 -0700
Subject: patch: update the build distro summary info

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 06a1a17..21d7439 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,8 +42,8 @@ set(GFXRECONSTRUCT_PROJECT_VERSION_MAJOR 0)
 set(GFXRECONSTRUCT_PROJECT_VERSION_MINOR 9)
 set(GFXRECONSTRUCT_PROJECT_VERSION_PATCH 8)
 
-set(GFXRECON_PROJECT_VERSION_DESIGNATION "-unknown")
-set(GFXRECON_PROJECT_VERSION_SHA1 "unknown-build-source")
+set(GFXRECON_PROJECT_VERSION_DESIGNATION "")
+set(GFXRECON_PROJECT_VERSION_SHA1 "bionic")
 
 if (NOT DEFINED HEADLESS)
     set(HEADLESS TRUE)
